src/Symfony/Component/Mailer/composer.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "symfony/mailer",
+    "type": "library",
+    "description": "Symfony Mailer Component",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3",
+        "psr/log": "~1.0",
+        "symfony/event-dispatcher": "^4.3",
+        "symfony/mime": "^4.3"
+    },
+    "require-dev": {
+        "symfony/amazon-bridge": "^4.3",
```

Comment:
In the composer.json file of the Amazon Bridge, the package is named `amazon-mailer`. Also, given that Amazon is much more than a mailer backend, the name `amazon-bridge` might be too generic for what the package actually provides.

Reply:
fixed